app/core/vueRouter.coco.js

```diff
@@ -100,6 +100,16 @@ export default function getVueRouter () {
           path: '/teachers/resources_new',
           component: () => import(/* webpackChunkName: "teachers" */ 'app/views/teachers/teacher-dashboard/BaseResourceHub/index.vue'),
         },
+        {
+          path: '/podcast',
+          name: 'AllPodcasts',
+          component: () => import(/* webpackChunkName: "podcastMain" */'/app/views/podcast/PodcastHomeView')
+        },
+        {
+          path: '/podcast/:handle',
+          name: 'PodcastSingle',
+          component: () => import(/* webpackChunkName: "podcastSingle" */'/app/views/podcast/SinglePodcastView')
+        }
```

Comment:
We might want to add to vueRouter.ozar.js, too, just so that the file isn't diverging even more. (Ideally we would be making progress towards merging these together and not having separate router files.) If we want ozaria.com/podcast to not work, we could put a redirect to codecombat.com/podcast in the application logic somewhere, depending on the `isOzaria` flag. Same thing for other .coco-specific additions.